deploy/tutorials/static-site.md

```diff
@@ -39,7 +39,7 @@ You have now a html page that says "Hello" and has a logo.
 To deploy this repo on Deno Deploy, from the `static-site` repository, run:
 
 ```console
-deployctl deploy --project=<your-preferred-project-name> jsr:@std/http@1.0.0-rc.5/file-server
+deployctl deploy --project=<your-preferred-project-name> https://jsr.io/@std/http/1.0.0-rc.5/file_server.ts
```

Comment:
can use use https://jsr.io/@std/http/1.0.6/file_server.ts

Reply:
I don't think so, it will only be fixed in 1.0.7, which isn't out yet…

Reply:
1.0.7 has the fix: https://github.com/denoland/std/pull/6033#issuecomment-2370906124 !